signadot/operator/templates/resources.signadot.com-customresourcedefinition.yaml

```diff
@@ -7,6 +7,7 @@ metadata:
     {{- range $key, $val := .Values.commonAnnotations }}
     {{ $key | quote }}: {{ $val | quote }}
     {{- end }}
+  creationTimestamp: null
```

Comment:
Hmm, wondering why this change crept in - any dependency of the updater script?

Reply:
I don't think it is a function of the change to hack/templatize.go.  I would guess it's some toolchain version diff.